src/vsphere_cpi/Gemfile

```diff
@@ -20,4 +22,10 @@ group :development, :test do
   gem 'rspec-its'
   gem 'simplecov', :require => false
   gem 'timecop', '~>0.8.1'
+  gem 'vcr', '~> 3.0', '>= 3.0.1'
+  gem 'webmock', '~> 1.24', '>= 1.24.3'
+  gem 'autotest', '~> 4.4', '>= 4.4.6'
+  gem 'autotest-rails-pure', '~> 4.1', '>= 4.1.2'
```

Comment:
What's a Rails gem doing in here? In fact do we need autotest at all?

Reply:
This is again required for nsxt client.